### ISSUE TITLE
Visual block do no span across empty lines

### ISSUE THREAD

**Issue by gerardroche** [NeoVintageous/NeoVintageous#18]
<a href="https://github.com/rodcloutier"><img src="https://avatars3.githubusercontent.com/u/97381?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [rodcloutier](https://github.com/rodcloutier)**
_Wednesday Jul 24, 2013 at 15:40 GMT_
_Originally opened as https://github.com/guillermooo/Vintageous/issues/295_

----

In visual block with the following (no space on the empty line)

```
foo
foo

foo
```

`ljjj` and `l3j` result in

```
[fo|o
|fo]o

foo
```

If you add spaces on the empty line:

`ljjj`

```
[fo|o
|fo|
|  |
|fo]o
```

`l3j`

```
[fo|o
|fo]

foo
```

**Comment by gerardroche** [comment]
<a href="https://github.com/m-o-e"><img src="https://avatars1.githubusercontent.com/u/1622080?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [m-o-e](https://github.com/m-o-e)**
_Friday Dec 05, 2014 at 01:04 GMT_

----

bump!

**Comment by gerardroche** [comment]
<a href="https://github.com/nolanpro"><img src="https://avatars2.githubusercontent.com/u/2546850?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [nolanpro](https://github.com/nolanpro)**
_Thursday Jan 08, 2015 at 23:48 GMT_

----

+1 (thanks for all your work btw)

**Comment by gerardroche** [comment]
<a href="https://github.com/kevinwheeler"><img src="https://avatars0.githubusercontent.com/u/5334864?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [kevinwheeler](https://github.com/kevinwheeler)**
_Monday Jun 29, 2015 at 23:31 GMT_

----

what nolanpro said.

**Comment by gerardroche** [comment]
<a href="https://github.com/kevinwheeler"><img src="https://avatars0.githubusercontent.com/u/5334864?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [kevinwheeler](https://github.com/kevinwheeler)**
_Monday Jun 29, 2015 at 23:37 GMT_

----

There are a few things that make this less than ideal, but there is a workaround for some use cases. When you hit the empty line, command+click (on mac, I assume control+click would work otherwise) the empty line and the one after it, and then keep going on as you were.

**Comment by gerardroche** [comment]
@rodcloutier @m-o-e @nolanpro @kevinwheeler @pmalek @drewcassidy @normenmueller @YayConnolly @mzhang001 @ghking1 @MSch Visual blocks have been rewritten and many issues have be resolved in recent versions; v1.14.1 at time of writing (fyi NeoVintageous is a fork of the discontinued Vintageous).
